cache/disk/findmissing.go

```diff
@@ -88,21 +119,27 @@ func filterNonNil(blobs []*pb.Digest) []*pb.Digest {
 // of missing blobs.
 func (c *diskCache) findMissingLocalCAS(blobs []*pb.Digest) int {
 	var exists bool
+	var item lruItem
 	var key string
 	missing := 0
 
 	c.mu.Lock()
 
 	for i := range blobs {
-		if blobs[i].SizeBytes == 0 {
+		if blobs[i].SizeBytes == 0 && blobs[i].Hash == emptySha256 {
```

Comment:
Just curious, did you hit a client bug here?

Reply:
No, simply trying to make it consistent with https://github.com/buchgr/bazel-remote/blob/e0f5a5c447e591949d05da040b653e89f9ce8d1d/cache/disk/disk.go#L1029